modules/dynamic-plug-in-development.adoc

```diff
@@ -19,12 +19,9 @@ Red{nbsp}Hat does not support custom plugin code. Only link:https://access.redha
 ifndef::openshift-rosa-hcp,openshift-rosa[]
 an {product-title} 
 endif::openshift-rosa-hcp,openshift-rosa[]
-ifdef::openshift-rosa-hcp[]
-a {rosa-short} 
-endif::openshift-rosa-hcp[]
-ifdef::openshift-rosa[]
-a {rosa-classic-short} 
-endif::openshift-rosa[]
+ifdef::openshift-rosa-hcp,openshift-rosa[]
+a {product-title} 
+endif::openshift-rosa-hcp,openshift-rosa[]
```

Comment:
In the module below this one, this need a space between the two words:
<img width="901" height="227" alt="image" src="https://github.com/user-attachments/assets/21409989-b559-4f87-9084-7c48de180adc" />

Classic: https://98558--ocpdocs-pr.netlify.app/openshift-rosa/latest/web_console/dynamic-plugin/dynamic-plugins-reference#dynamic-plugin-api_dynamic-plugins-reference

HCP: https://98558--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/web_console/dynamic-plugin/dynamic-plugins-reference#dynamic-plugin-api_dynamic-plugins-reference